lua/SS13_base.lua

```diff
@@ -19,12 +19,46 @@ function SS13.get_runner_client()
 	return dm.global_vars.GLOB.directory[SS13.get_runner_ckey()]
 end
 
+SS13.ispath = dm.global_procs._ispath
```

Comment:
Because this returns 1/0, it's a trap because that's not how Luau truthiness works. This should be wrapped up in a function that converts it back to a bool.

Reply:
oh true lol